docs/content/configuration/coordinator.md

```diff
@@ -22,6 +22,7 @@ The coordinator node uses several of the global configs in [Configuration](../co
 
 |Property|Description|Default|
 |--------|-----------|-------|
+|`druid.coordinator.beOverlord`|Boolean value for whether this coordinator node should act like an overlord as well. This configuration allows users to simplify a druid cluster by not having to deploy any standalone overlord nodes. If set to true, then make sure `druid.selectors.indexing.serviceName` is correctly set to coordinator service name everywhere and all the overlord configuration must be added to the coordinator node configuration.|true|
```

Comment:
I'm happy with beOverlord being false by default. At the same time we can set it to true in the quickstart configs and remove overlord from the quickstart. That gets us a lot of the simplicity benefit for new users without making life difficult for already existing deployments.